[Docs] Add link to dependencies page - 9.0 (#214668)
elastic/kibana: docs/extend/dependencies-versions.md

# List of dependencies [dependencies-versions]

> Refer to [Elastic Stack third-party dependencies](https://artifacts.elastic.co/reports/dependencies/dependencies-current.html) for a list of the external dependencies you can leverage.

Reviewer: ```suggestion
Refer to [Elastic Stack third-party dependencies](https://artifacts.elastic.co/reports/dependencies/dependencies-current.html) for the complete list of dependencies.
```
